specs/pipeline/edit/GeneralSettings.spec

```diff
@@ -0,0 +1,25 @@
+// --GO-LICENSE-START--
```

Comment:
After talking to legal — they also mentioned that this license header is NOT needed in the spec files. Just the java source is ok.